autocannon.js

```diff
@@ -239,9 +239,17 @@ function start (argv) {
       })
     })
   } else {
-    initJob(argv).catch((err) => {
+    // if forever is true then a promise is not returned and we need to try ... catch errors
```

Comment:
Question: if rather than handling this in a special way we made the forever code path return a promise too, so that we don't have to handle special cases here?

Reply:
This was my original approach but I ran into problems with the reporting. I just tried again now and it is working 😕 . Would you like me to investigate further or shall I merge as is?

Reply:
Up to you. My personal preference would be to do it "properly", but I'm not a maintainer of the repo either so I leave that to you and the maintainers.